test-provider-sample/package.json

```diff
@@ -22,7 +22,8 @@
 		"compile": "tsc -p ./",
 		"lint": "eslint \"src/**/*.ts\"",
 		"watch": "tsc -watch -p ./",
-		"postinstall": "npm run download-api"
+		"postinstall": "npm run download-api",
+		"download-api": "npx @vscode/dts dev"
```

Comment:
We actually don't depend on dev typings any more, so both the postinstall and download-api scripts can be deleted

Reply:
fixed